termwiz/src/input.rs

```diff
@@ -1267,6 +1267,21 @@ impl InputParser {
                                         modifiers,
                                     }));
                                 }
+                                // AZL - I don't quite understand this
```

Comment:
Ah, this is for terminal applications who are listening for mouse reports from the terminal.
I think it's important for the application to know that the x/y are pixels for this case.
Perhaps the simplest way to encode that is to introduce a `InputEvent::PixelMouse(MouseEvent)` variant further up in this file:

https://github.com/wez/wezterm/blob/main/termwiz/src/input.rs#L63-L66

and then switch to `PixelMouse` in the callback we issue below?

Reply:
How does this look now?